src/api/visualizer.coffee

```diff
@@ -1,36 +1,33 @@
 Event = require('../model/events').Event
```

Comment:
Oh, if it's deprecated maybe we should just remove it entirely. The next release is going to be a breaking one anyways.